### PR TITLE
E-768 refactor: do not show recovery for otp without phone

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

**Comment by ifiri** [comment]
@kazurus approved

File: src/components/screens/Authenticator/Authenticator.interactor.vue [hunk 1]
```diff
@@ -43,6 +47,13 @@ export default {
       },
       immediate: true,
     },
+
+    isPasswordExist: {
```
Comment (kazurus) [inline src/components/screens/Authenticator/Authenticator.interactor.vue:51]: Why do we need this?

Comment (kazurus) [reply src/components/screens/Authenticator/Authenticator.interactor.vue:51]: If we need it on higher level we should pass it with submit

**Pull request merged**